pkg/operator/ceph/object/objectstore.go

```diff
@@ -99,59 +105,79 @@ func reconcileRealm(context *Context, serviceIP string, port int32) error {
 		defaultArg = "--default"
 	}
 
-	// create the realm if it doesn't exist yet
-	output, err := runAdminCommand(context, "realm", "get")
-	if err != nil {
+	// obviously need to change this
+	if isMultisite(spec) {
 		updatePeriod = true
```

Comment:
nit: For code readability I'd suggest helper methods for multisite or non-multisite code blocks. Otherwise the method gets too long.

Reply:
This is one of the items I'd like to address in a future PR.